main.py

```diff
@@ -59,6 +58,31 @@ def on_keypress(bit_index):
   clock.tick(60)
   pygame.display.flip()
 
+  if not game.state_of_game:
+    draw_layout(game, binary_boxes, binary_bar_preview, score_display)
+    for event in pygame.event.get():
+      match event.type:
+        case pygame.QUIT:
+          pygame.quit()
+          exit()
+        case pygame.KEYDOWN:
+          if event.key == pygame.K_SPACE:
+            game.state_of_game = True
+            draw_layout(game, binary_boxes, binary_bar_preview, score_display)
```

Comment:
You're redrawing everything for a second time in one frame just to get rid of the box in the middle of the screen?